### PR TITLE
ads

### PR DESCRIPTION
**Pull request by raulriera**
more tesina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,199 @@
-TEST
+Hello darkness, my old friend
```
Comment (dannbeaus) [inline README.md]: https://www.youtube.com/watch?v=KQOokYw8174

**Comment by raulriera** [comment]
Approve it and let's see if it really works 😜

**Comment by raulriera** [comment]
@krbarnes leaving us in suspense

**Comment by dannbeaus** [comment]
I am able to merge before that approval; what of the minimal required approvals?

**Comment by raulriera** [comment]
I don't want to prevent people from merging... it should just be an advice

**Comment by dannbeaus** [comment]
https://github.com/apps/minimun-reviews is it though?

**Comment by dannbeaus** [comment]
did @raulriera create this bot? 

~probot~ paulbot :trollface:

**Comment by krbarnes** [comment]
Lo rompi

screen shot 2018-04-07 at 8 56 56 pm

**Comment by raulriera** [comment]
You broke it Kevin... and yes I wrote it 😜

**Comment by dannbeaus** [comment]
I cannot see the probot app by the way, when I click details it's private

**Comment by dannbeaus** [comment]
> that spelling mistake tho...
I always `minimun` instead of `minimum`

it's my affinity towards `nn`

**Comment by raulriera** [comment]
Its all in my pc at the moment... and damn, I wasn't listening for the `pull_request_review.submitted` events 😬 just stuff from the one making the PR should be fixed now

**Comment by dannbeaus** [comment]
amazing!

**Pull request closed without merging**